src/components/staking/StakingContainer.js

```diff
@@ -185,13 +185,19 @@ export function StakingContainer({ history, match }) {
                 async () => {
                     await dispatch(stake(currentAccount.accountId, validator, amount))
                     Mixpanel.people.set({last_stake_time: new Date().toString()})
+                },
```

Comment:
@Patrick1904 let's make this default error handler if none is passed into `withTracking`?

allows to fix all similar errors